docs/orchestration/concepts/automations.md

```diff
@@ -8,29 +8,212 @@ You can set up and edit your Automations using the [API](/orchestration/concepts
 
 ## Events
 
-An _event_ is a configurable condition that, when met, will cause an automation to run. Currently you can configure automations for the following events:
-- A flow run enters a given state or states
-- Runs from multiple (or all!) flows enter a given state or states
+The automations system is composed of events and actions. When an _event_ is paired with an _action_, we call it an _automation_.
```

Comment:
This is a very nice definition. I think it belongs in the Automation section above.

Reply:
Thanks! Updated the intro section